### PR TITLE
Release 1.4

### PR DESCRIPTION
**Pull request by clover-bret**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Version 
 
-Current version: 1.3.2
+Current version: 1.4
```
Comment (mike-brooking) [inline README.md]: Should we be using a full 3 part version number, i.e. 1.4.0 ?

**Review by mike-brooking — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mike-clover-com) [reply README.md]: Yes.

File: remote-pay-android-connector/build.gradle [hunk 2]
```diff
@@ -1,18 +1,17 @@
 group = 'com.clover.sdk'
-version = '1.3.2'
+version = '1.4'
```
Comment (mike-brooking) [inline remote-pay-android-connector/build.gradle]: Same on version number

**Review by mike-brooking — APPROVED** [review]

**Review by mike-clover-com — CHANGES_REQUESTED** [review]
Just make the change Mike Brooking indicated please.  1.4.0

**Review by mike-clover-com — APPROVED** [review]

**Comment by clover-bret** [comment]
Corrected version numbers to reflect semantic versioning

**Pull request merged**